setup.py

```diff
@@ -120,7 +120,10 @@ def build_extension(self, ext):
     packages=find_packages("src"),
     package_dir={"": "src"},
     include_package_data=True,
-    install_requires=["jax", "jaxlib"],
+    install_requires=[
+        "jax==0.4.19",
+        "jaxlib==0.4.19"
+    ],
```

Comment:
I'd rather not pin to a specific version. Could we set a lower bound instead? Then we'll just aim to keep this updated as the interface continues to change.